### PR TITLE
scalarxmat44

### PR DESCRIPTION
**Pull request by gtsoul-tech**

### CODE REVIEW

File: scalarxmat44.c [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    // fill array with numbers
+    for (int i = 0; i < 4; i++) {
+        for (int j = 0; j < 4; j++) {
+            A[i][j] = (float) count++;
+        }
+    }
+    count = 0;
+    for (int i = 0; i < 4; i++) {
+        for (int j = 0; j < 4; j++) {
+            B[i][j] = (float) count++;
```
Comment (markos) [inline scalarxmat44.c:35]: I`d suggest B to have different kind of elements, eg. use a random or the reverse index, or count * constant.

Comment (gtsoul-tech) [reply scalarxmat44.c:35]: Ι used A to store the result of scalar * A4x4 and B4x4 to store the implementation of sse,avx2....
and then I compare the matrices

Comment (markos) [reply scalarxmat44.c:35]: oh sorry this is not for matrix multiplication, ignore

**Review by markos — APPROVED** [review]
LGTM

**Pull request merged**